tests/models/test_regression.py

```diff
@@ -25,33 +26,59 @@ def coef_(self):
 
         monkeypatch.setattr(LinearRegression, 'train', mocktrain)
 
-        model = LinearRegression(tmp_path)
+        model = LinearRegression(tmp_path, experiment='one_month_forecast')
         model.train()
         model.save_model()
 
-        assert (tmp_path / 'models/linear_regression/model.npy').exists(), f'Model not saved!'
+        assert (
+            tmp_path / 'models/one_month_forecast/linear_regression/model.npy'
+        ).exists(), f'Model not saved!'
 
-        saved_model = np.load(tmp_path / 'models/linear_regression/model.npy')
+        saved_model = np.load(tmp_path / 'models/one_month_forecast/linear_regression/model.npy')
         assert np.array_equal(model_array, saved_model), f'Different array saved!'
 
-    @pytest.mark.parametrize('use_pred_months', [True, False])
-    def test_train(self, tmp_path, capsys, use_pred_months):
+    @pytest.mark.parametrize('use_pred_months,experiment',
+                             [(True, 'one_month_forecast'),
+                              (True, 'nowcast'),
+                              (False, 'one_month_forecast'),
+                              (False, 'nowcast')])
+    def test_train(self, tmp_path, capsys, use_pred_months, experiment):
         x, _, _ = _make_dataset(size=(5, 5), const=True)
         y = x.isel(time=[-1])
 
-        test_features = tmp_path / 'features/train/hello'
+        test_features = tmp_path / f'features/{experiment}/train/hello'
         test_features.mkdir(parents=True)
-
-        norm_dict = {'VHI': {'mean': np.zeros(x.to_array().values.shape[:2]),
-                             'std': np.ones(x.to_array().values.shape[:2])}
-                     }
-        with (tmp_path / 'features/normalizing_dict.pkl').open('wb') as f:
+        pred_features = tmp_path / f'features/{experiment}/test/hello'
+        pred_features.mkdir(parents=True)
+
+        if experiment == 'nowcast':
```

Comment:
Same comment as above

Reply:
the nowcast requires other variables though becuase otherwise the `current` array will be empty